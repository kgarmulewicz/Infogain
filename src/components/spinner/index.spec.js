import "@testing-library/jest-dom/extend-expect";
import React from "react";
import { render } from "@testing-library/react";
import Spinner from './index';

describe("<Spinner />", () => {
  test("Spinner match snapshot!", () => {
    const { view } = render(<Spinner />);
    expect(view).toMatchSnapshot();
  });
});
