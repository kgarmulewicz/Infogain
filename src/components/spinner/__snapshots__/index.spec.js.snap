// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Spinner match snapshot! 1`] = `undefined`;
